.github/workflows/watch-conda-forge.yml

```diff
@@ -0,0 +1,37 @@
+name: Check for new versions of pangeo-notebook metapackage on Conda Forge
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    if: github.repository == 'pangeo-data/pangeo-stacks-dev'
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get latest pangeo-notebook version
+        id: latest_version
+        uses: jacobtomlinson/gha-anaconda-package-version@0.1.1
+        with:
+          org: "conda-forge"
+          package: "pangeo-notebook"
+      - name: Find and Replace Base
+        uses: jacobtomlinson/gha-find-replace@0.1.1
+        with:
+          include: "*notebook/"
+          find: "pangeo-notebook=.* "
```

Comment:
Looks like this searches all plaint text files in each notebook subdirectory?

Reply:
that's right.

Reply:
I've updated this to include only environment.yml files.